veocean-data-farming/veocean.md

```diff
@@ -0,0 +1,111 @@
+
+The amount of reward depends on how long the tokens are locked for. You must lock up your OCEAN into the vault for to obtain veOCEAN. Going forward, veOCEAN will be the main mechanism for staking OCEAN, and for curation of datasets.
+
+After creating your lock you will be credited veOCEAN. We sometimes refer to veOCEAN as your “voting power”.
+
+**WARNING:** You will not be able to retrieve your original OCEAN deposit until the lock ends.
+
+## What can I do with veOCEAN?
+
+veOCEAN allows you to engage with different protocol mechanisms and benefit from the reward programs available.
+
+There are 4 things you can do with veOCEAN.
+1. **Hold it** veOCEAN pays **Passive Rewards** every week.
+2. **Allocate it** veOCEAN pays **Active Rewards** every week to the top performing Datasets, Algorithms, dApps, and more.
+3. **Delegate it** You can delegate veOCEAN to other Data Farmers who can curate Datasets for you. In return for their services, these farmers may charge you a fee for helping you receive APY on **Active Rewards**. The Delegate feature has just been recently released and enables veOCEAN holders to more easily access Active Rewards.
+4. **2x Stake** If you are a publisher, allocating veOCEAN to your own Dataset gives your veOCEAN a 2x Bonus. This is an incentive for publishers to engage with their assets and benefit from from the protocol further.
+
+## What is time locking?
+
+Users can lock their OCEAN for different lengths of time to gain voting power. Our app is configured to lock OCEAN for a minimum of 2 weeks and a maximum of four years for max benefit.
+
+Users that lock their OCEAN for a longer period of time receive more veOCEAN to reflect their conviction in the system.
+
+| Year | Lock Multiplier | veOCEAN |
+| ---- | ----------| ------- |
+| 1    | 0.25x     | 0.25    |
+| 2    | 0.50x      | 0.50     |
+| 3    | 0.75x     | 0.75    |
+| 4    | 1.00x      | 1.00     |
+
+_The Lock Multiplier. Amount of veOCEAN received per OCEAN locked._
+
+If you’ve locked OCEAN for 4 years, you will be unable to retrieve your deposit until this time expires.
+
+After choosing your lock period and locking up your OCEAN into the vault, you will be credited with veOCEAN.
+
+veOCEAN is non-transferable. You can’t sell it or send it to other addresses.
+
+## Linear Decay
+
+Your veOCEAN balance will slowly start declining as soon as you receive it.
+
+veOCEAN balance decreases linearly over time until the Lock End Date. When your lock time has lapsed by 50%, you will have 50% of your original veOCEAN balance.
+
+When your lock time ends your veOCEAN balance will hit 0, and your OCEAN tokens can be withdrawn.
+
+If you lock 1.0 OCEAN for 4 years, you get 1.0 veOCEAN at the start.
+
+| Years Passed | veOCEAN Left |
+| ---- | ---- |
+| 1 year | 0.75 |
+| 2 years | 0.50 |
+| 3 years | 0.25 |
+| 4 years | 0.00 |
+
+At the end of your 4 years, your OCEAN is unlocked. [1]
```

Comment:
What's this [1] for?

Reply:
From [this blog post](https://blog.oceanprotocol.com/veocean-is-launching-data-farming-is-resuming-abed779211e3)

```
[1] veOCEAN held decays linearly over time. You can calculate the balance as follows: veOcean_balance = OCEAN_amount_locked * (your_unlock_timestamp — current_unix_timestamp ) / 60 * 60 * 24 * 7 * 52 (that is 4 years)
```

I have avoided adding math functions and metric calculations. I'll be expanding on these in the future.

Reply:
Resolved.